gig-marketplace.md

```diff
@@ -0,0 +1,58 @@
+## Title
+* Create an intranet marketplace where InnerSource project owners and potential contributors can easily discover one another
```

Comment:
I think that generally this title is much shorter - using the same words as the file name.  The "normal" title for this document would be something like: **Gig Marketplace**.

Reply:
Great point Russ.  "Gig Marketplace" it is. 👍